qemu/tests/cfg/qemu_disk_img_compare.cfg

```diff
@@ -0,0 +1,26 @@
+- qemu_disk_img_compare:
+    virt_test_type = qemu
+    type = qemu_disk_img_convert
+    kill_vm = yes
+    start_vm = no
+    image_size = 20G
+    force_create_image = no
+    backup_image_before_testing = yes
+    restore_image_before_testing = yes
+    option_verified = "format"
+    image_convert = "image1"
+    convert_name_image1 = "images/image1_to_raw"
+    convert_format_image1 = "raw"
+    compare_mode_list = "False True"
```

Comment:
True  False is not clear enough, suggest to update compare_mode_list to strict_mode